hub/powertoys/mouse-without-borders.md

```diff
@@ -82,6 +82,23 @@ To enable the service mode, run PowerToys in administrator mode and turn on the
 | Add a firewall rule for Mouse Without Borders | Install a firewall rule for Mouse Without Borders. |
 | Show the original Mouse Without Borders UI | Show the original UI from Mouse Without Borders through the original tray icon. Mouse Without Borders needs to be restarted for it to take effect. |
 
+## Status Colors
+The following colors are used to indicate the connection status to the user when trying to connect to another computer:
+
+| Connection Status | Color    | Hex Code    |
+| :-----: | :---: | :---: |
+| NA | Dark Grey   | #00717171  |
```

Comment:
```suggestion
| NA | Dark Grey | `#00717171` |
```
Formatting the hex codes as code looks better. Could you do this for each row?

Reply:
Added now!